### PR TITLE
prefix all componentId with `sc-`

### PR DESCRIPTION
**Pull request by robilars-amazon**
Previously componentId would only be prefixed with `sc-` when component hash began with a digit. This differs from styled-components componentId generation where all componentIds are prefixed with `sc-`

This patch causes all componentId to be prefixed with `sc-`

Additionally, I have updated the build dependancies and rebuilt yarn.lock

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,18 +15,18 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.4.4",
-    "@babel/core": "^7.4.5",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.8.7",
     "@babel/plugin-proposal-class-properties": "^7.4.4",
-    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-env": "^7.8.7",
```
Comment (probablyup) [inline package.json:21]: please drop unrelated changes, thanks

**Comment by tobilen** [comment]
i believe this can be closed, since there is the `namespace' option that parameterizes this now

**Comment by kitten** [comment]
I believe this is obsolete now given: https://github.com/styled-components/babel-plugin-styled-components/pull/281
